wiki/Main_Page/es.md

```diff
@@ -42,14 +39,14 @@ Todo sobre círculos, tambores, frutas y el sintetizador: elementos clave, conce
 
```

Comment:
While we are here, I would change the `Juego` title to `Jugabilidad` as its a better translation for `Gameplay`